data/hai/hai news.txt

```diff
@@ -421,20 +421,20 @@ news "hai joining unfettered"
 		government "Hai (Unfettered)"
 	name
 		word
-			"Hai merchant (sympathiser)"
+			"Hai merchant"
```

Comment:
This one was said to be a sympathizer on purpose bexause they are the only ones bringing Unfettered to their space, but I guess that since it happens on an unfettered world it should be fairly obvious